flextape/server/BUILD.bazel

```diff
@@ -18,5 +19,40 @@ go_library(
 go_binary(
     name = "flextape",
     embed = [":go_default_library"],
-    visibility = ["//visibility:public"],
+)
+
+pkg_tar(
+    name = "flextape_tar",
+    srcs = [":flextape"],
+    package_dir = "/enfabrica/bin",
+)
+
+pkg_tar(
+    name = "flextape_config_tar",
+    srcs = ["flextape_config.textproto"],
+    package_dir = "/enfabrica/config",
```

Comment:
This is okayish for now, usually config files (if they're in the container) and labelled under /default or smth. Usually, they're just documented

Reply:
ack